idea/src/org/jetbrains/kotlin/idea/intentions/ConvertBlockCommentToLineCommentIntention.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.psi.PsiComment
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiWhiteSpace
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.psi.KtPsiFactory
+
+class ConvertBlockCommentToLineCommentIntention : SelfTargetingIntention<PsiComment>(
```

Comment:
Should be probably not applicable to documentation comments:

```
/**
 * ...
 */
```

Reply:
it's enough to check that beginning of comment is `/**`

Reply:
I added `docComment.kt` test. `isBlockComment()` function returns false, if element is KDoc.